config/routes.rb

```diff
@@ -7,7 +7,11 @@
 
   get 'privacy' => 'static#privacy'
 
-  resources :users
+  resources :users do
+    member do
+      get 'dashboard'
+    end
+  end
```

Comment:
As mentioned in previous comments, this may not be necessary if we are collapsing `dashboard` into `users/show`